### PR TITLE
FAQ entry for Kafka / num docs

### PR DESCRIPTION
**Pull request by kkraune**
FYI @buinauskas

### CODE REVIEW

**Review by buinauskas — APPROVED** [review]
I'm flattered 🙏🏻

File: en/faq.md [hunk 1]
```diff
@@ -201,6 +201,9 @@ be searched using the query api, but can be used by [visiting](visiting.html).
 #### Visiting does not dump all documents, and/or hangs.
 There are multiple things that can cause this, see [visiting troubleshooting](visiting.html#troubleshooting).
 
+#### How to find number of documents in the index?
+Use metrics or `vespa visit` - see [examples](/en/operations/batch-delete.html#example).
```
Comment (jobergum) [inline en/faq.md:205]: This is rather expensive compared to running a query for the document types no?

Comment (kkraune) [reply en/faq.md:205]: true - I'll add a query example, too

**Comment by jobergum** [comment]
:100

**Pull request merged**